### PR TITLE
Implement parts of LegoJetskiRaceActor

### PR DESCRIPTION
**Pull request by jonschz**
The new functions match to 100 % and also match BETA10 structurally (when available).

### CODE REVIEW

File: LEGO1/lego/legoomni/src/entity/legojetskiraceactor.cpp [hunk 1]
```diff
@@ -11,17 +22,120 @@ LegoJetskiRaceActor::LegoJetskiRaceActor()
...
 }
 
-// STUB: LEGO1 0x10081550
+// FUNCTION: LEGO1 0x10081550
 void LegoJetskiRaceActor::VTable0x70(float p_float)
 {
-	// TODO
+	if (m_unk0x0c == 0) {
+		const LegoChar* raceState = VariableTable()->GetVariable(g_raceState);
+		if (stricmp(raceState, g_racing) == 0x0) {
```
Comment (foxtacles) [inline LEGO1/lego/legoomni/src/entity/legojetskiraceactor.cpp:126]: I think it's clearer to just write it as `!stricmp(raceState, g_racing)` (the way it's done in other places)

File: LEGO1/lego/legoomni/src/entity/legojetskiraceactor.cpp [hunk 2]
```diff
@@ -11,17 +22,120 @@ LegoJetskiRaceActor::LegoJetskiRaceActor()
...
+		if (m_state == 1) {
+			if (m_userNavFlag) {
+				m_unk0xe4 = 0.5f;
+			}
+
+			v1 = m_destEdge->CCWVertex(*m_boundary);
+			v2 = m_destEdge->CWVertex(*m_boundary);
+			assert(v1 && v2);
+
+			a[0] = (*v1)[0] + ((*v2)[0] - (*v1)[0]) * m_unk0xe4;
```
Comment (foxtacles) [inline LEGO1/lego/legoomni/src/entity/legojetskiraceactor.cpp:88]: This looks like `LERP3` in `vec.h`

File: LEGO1/lego/legoomni/src/entity/legojetskiraceactor.cpp (hunk 1 above)
Comment (jonschz) [reply LEGO1/lego/legoomni/src/entity/legojetskiraceactor.cpp:126]: Funny, I personally find the comparison with zero clearer, but that's a matter of opinion (to me, `!stricmp` feels like it says "does not match" while meaning the opposite). I noticed that the rest of the repo isn't entirely consistent there either, but the exclamation mark pattern is much more prevalent. I'll change it.

File: LEGO1/lego/legoomni/src/entity/legojetskiraceactor.cpp (hunk 2 above)
Comment (jonschz) [reply LEGO1/lego/legoomni/src/entity/legojetskiraceactor.cpp]: This has had a negative effect on the stack match in BETA10, but the refactor definitely looks cleaner and LEGO1 is unaffected.

**Pull request merged**